data/core.yaml

```diff
@@ -219,14 +219,41 @@ en:
       multiple_ways: There are too many lines here to split.
       connected_to_hidden: This can't be split because it is connected to a hidden feature.
     restriction:
-      help:
-        select: Click to select a road segment.
-        toggle: Click to toggle turn restrictions.
-        toggle_on: 'Click to add a "{restriction}" restriction.'
-        toggle_off: 'Click to remove the "{restriction}" restriction.'
       annotation:
         create: Added a turn restriction
         delete: Deleted a turn restriction
+  restriction:
+    controls:
+      distance: Distance
+      distance_up_to: "Up to {distance}"
+      via: Via
+      via_node_only: "Node only"
+      via_up_to_one: "Up to 1 way"
+      via_up_to_multiple: "Up to {num} ways"
+    help:
+      no: NO
+      only: ONLY
+      allowed: Allowed
+      indirect: "(indirect)"
+      turn:
+        left_turn: Left Turn
+        right_turn: Right Turn
+        u_turn: U-Turn
+        straight_on: Straight On
+      no_turn_string: "{no} {turn} {indirect}"
+      only_turn_string: "{only} {turn} {indirect}"
+      allowed_turn_string: "{turn} {allowed} {indirect}"
```

Comment:
In some languages, it may be difficult to translate these `turn` strings independently of the overall sentence. A more sound approach would be to multiply out these strings by all the possible combinations, e.g., “ONLY Left Turn”. In the meantime, languages that have problems with these strings may be able to use the imperative form, e.g., “ONLY Turn Left”.

Reply:
Good point.. I'll just enumerate all of these (and leave the `{indirect}` as a token replacement)